integrations/haproxy/documentation.yaml

```
@@ -1,27 +1,36 @@
-exporter_type: sidecar
+exporter_type: included
 app_name_short: HAProxy
 app_name: {{app_name_short}}
 app_site_name: HAProxy
 app_site_url: https://www.haproxy.com/
-exporter_name: the HAProxy exporter
-exporter_pkg_name: haproxy_exporter
-exporter_repo_url: https://github.com/prometheus/haproxy_exporter
+exporter_name: the HAProxy promex exporter
```

```suggestion
exporter_name: the PROMEX service for HAProxy
```

Reading through https://github.com/haproxy/haproxy/blob/master/addons/promex/README and I think that's a more accurate name (they don't ever use "promex" in lowercase).